autoscheduler/scraper/tests/api_tests.py

```diff
@@ -28,10 +28,23 @@ def setUpTestData(cls):
                    title='Database Systems', term='201731', credit_hours=3),
             Course(id='CSCE315-201731', dept='CSCE', course_num='315',
                    title='Programming Studio', term='201731', credit_hours=3),
+            Course(id='ASCC101-201911', dept='ASCC', course_num=101,
+                   title='APPL OF LEARNING THEORY', term=201911, credit_hours=0),
+            Course(id='ASCC101-201931', dept='ASCC', course_num=101,
+                   title='APPL OF LEARNING THEORY', term=201931, credit_hours=0),
+            Course(id='ASCC101-201831', dept='ASCC', course_num=101,
+                   title='APPL OF LEARNING THEORY', term=201831, credit_hours=0),
+            Course(id='CSCE310-201911', dept='CSCE', course_num=310,
+                   title='DATABASE SYSTEMS', term=201911, credit_hours=3),
+            Course(id='BIMS110-201831', dept='BIMS', course_num=110,
+                   title='ONE HEALTH IN ACTION', term=201831, credit_hours=1),
         ]
         cls.instructors = [
             Instructor(id='Akash Tyagi'),
             Instructor(id='John Moore'),
+            Instructor(id='Morgan W. Jones'),
+            Instructor(id='Ronald Ward'),
+            Instructor(id='Colin Young'),
```

Comment:
The courses already in this are used by the course serializer tests, but I should move them into the course serializer tests themselves rather than the setup. You should just remove the ones you added and I'll refactor mine later to make the courses inside of the tests